openprescribing/templates/base.html

```diff
@@ -90,12 +90,68 @@
             <span class="icon-bar"></span>
           </button>
           <a class="navbar-brand" href="{% url 'home' %}">
-          <img class="logo" src="{% static 'img/logo.svg' %}" alt="OpenPrescribing logo" onerror="this.src='{% static "img/logo.png" %}';this.onerror=null;" />
-          OpenPrescribing
+            <span class="sr-only">OpenPrescribing</span>
+            <svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" fill-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2" clip-rule="evenodd" viewBox="0 0 383 110">
```

Comment:
Do we have to inline this so we can style it? Is there any alternative?

Reply:
I would prefer it to be code rather than an `img` tag, because that is easier to style. Would it be best if I switched it to an include tag and made a HTML partial?